diesel/src/pg/expression/expression_methods.rs

```diff
@@ -1111,3 +1111,113 @@ impl InetOrCidr for Inet {}
 impl InetOrCidr for Cidr {}
 impl InetOrCidr for Nullable<Inet> {}
 impl InetOrCidr for Nullable<Cidr> {}
+
+/// PostgreSQL specific expression methods using the `Jsonb` datatype.
+pub trait PgJsonbExpressionMethods: Expression<SqlType = Jsonb> + Sized {
```

Comment:
To allow nullable types here we must lift that constraint a bit:
```suggestion
pub trait PgJsonbExpressionMethods: Expression + Sized {
```